riposte-core/src/test/java/com/nike/riposte/server/componenttest/VerifyProxyRequestCornerCasesComponentTest.java

```diff
@@ -364,7 +364,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                 if ("true".equals(state.getRequestInfo().getHeaders().get(INTENTIONAL_EXPLOSION_AFTER_LAST_CHUNK_HEADER_KEY))) {
                     ctx.channel().eventLoop().schedule(() -> {
                         ctx.fireExceptionCaught(new ApiException(INTENTIONAL_EXPLOSION_AFTER_LAST_CHUNK_API_ERROR));
-                    }, 1, TimeUnit.MILLISECONDS);
+                    }, 100, TimeUnit.MILLISECONDS);
```

Comment:
On travis CI this error was being processed before the last chunk was sent, causing the last chunk to trigger an error that prevented the test from working. Adding some extra buffer time seems to fix it.